src/generate.ml

```diff
@@ -33,6 +33,82 @@
 
 open Structures
 open Util
+open Graphlib.Std
```

Comment:
I have a small reservation around using Core/Core_kernel here because it is a heavy dependency that brings in many other dependencies. It is fine to keep as it is and I appreciate the fix @Nymphium.
Was there a particular feature of `graphlib` that couldn't be done with the original graph library.

Reply:
I wanted to use `Graphlib.strong_components` but `Graph.Components`  in ocamlgraph can achieve it. I rewrite using it.

Reply:
Thank you for rewriting it, keeping a small set of dependencies for the generator code is good.